<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ICP Data Validation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>RGB-ALS-LogoCircle-Blue@2x.jpg</normaloff>RGB-ALS-LogoCircle-Blue@2x.jpg</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_sootAbsorbance">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Soot Absorbance</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_abs">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sootPercentage">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Soot Percentage</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_prcnt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_convert">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
